src/controller/data_model/controller-clusters.zap

```diff
@@ -1232,7 +1232,7 @@
               "storageOption": "RAM",
```

Comment:
Is there a way to avoid storageOption: RAM for all the fixed values?

Reply:
https://github.com/project-chip/connectedhomeip/issues/5926 tracks that...